docs/setup/upgrade.asciidoc

```diff
@@ -76,3 +77,5 @@ include::upgrade/saved-objects-migration.asciidoc[]
 include::upgrade/resolving-migration-failures.asciidoc[]
 
 include::upgrade/rollback-migration.asciidoc[]
+
+include::{kibana-root}/docs/settings/logging-settings.asciidoc[]
```

Comment:
```suggestion
```
This is not needed, in fact it actually breaks the build 😅 
The link you added on line 10 is sufficient